cmd/kubeadm/app/phases/upgrade/compute_test.go

```diff
@@ -34,6 +34,10 @@ import (
 	etcdutil "k8s.io/kubernetes/cmd/kubeadm/app/util/etcd"
 )
 
+const (
+	coreDNSVersion = "1.3.1"
```

Comment:
Can we just use `constants.CoreDNSVersion` instead of declaring a new constant here?

Reply:
+1
Also, TIL about `constants.Foo`

Reply:
Done